source/reference/minio-cli/minio-mc/mc-sql.rst

```diff
@@ -230,6 +230,13 @@ Syntax
    - ``BZIP2``
    - ``NONE`` (default)
 
+   Compression schemes supported by MinIO backend only:
```

Comment:
Syntax is fine - one clarifying question. By `MinIO backend only`, do we mean that these commands only work if `mc sql` is targeting these files stored on MinIO vs another S3 storage?

Reply:
@ravindk89 Files stored on MinIO only.

Reply:
OK just checking - this is perfect, thank you for opening the pull :)